util/compression.h

```diff
@@ -333,17 +333,24 @@ class CompressionInfo {
   const CompressionContext& context_;
   const CompressionDict& dict_;
   const CompressionType type_;
+  const uint64_t sample_for_compression_;
```

Comment:
Is this necessary?

Reply:
This is the main place where it is stored -- from earlier discussions.

Reply:
sg